libs/application/templates/estate/src/fields/EstateMembersRepeater/AdditionalEstateMember.tsx

```diff
@@ -143,6 +148,33 @@ export const AdditionalEstateMember = ({
             required
           />
         </GridColumn>
+        {application.answers.selectedEstate ===
+          EstateTypes.permitForUndividedEstate && (
+          <GridColumn span={['1/1', '1/2']} paddingBottom={2}>
+            <SelectController
+              key={relationWithApplicantField}
+              id={relationWithApplicantField}
+              name={relationWithApplicantField}
+              label={formatMessage(m.inheritanceRelationWithApplicantLabel)}
+              defaultValue={field.relationWithApplicant}
+              options={relationWithApplicantOptions}
+              error={error?.relationWithApplicant}
+              backgroundColor="blue"
+              required={!field.initial}
+            />
+          </GridColumn>
+        )}
+        <GridColumn span={['1/1', '1/2']} paddingBottom={2}>
+          <InputController
+            id={emailField}
+            name={emailField}
+            label={m.email.defaultMessage}
+            defaultValue={field.email || ''}
+            backgroundColor="blue"
+            error={error?.email}
+            required
+          />
+        </GridColumn>
```

Comment:
Context: This is necessary because the zod validation on the base level interrupts the type in the application template.